SimFastTiming/FastTimingCommon/src/ETLElectronicsSim.cc

```diff
@@ -158,15 +161,20 @@ void ETLElectronicsSim::runTrivialShaper(ETLDataFrame& dataFrame,
     //ETLSample newSample;
     dataFrame.setSample(it, newSample);
 
-    if (debug)
-      edm::LogVerbatim("ETLElectronicsSim") << adc << " (" << chargeColl[it] << "/" << adcLSB_MIP_ << ") ";
+#ifdef EDM_ML_DEBUG
```

Comment:
@rdelliga , why both EDM_ML_DEBUG and debug? I would use one or another.

Reply:
@fabiocos , it is not a strong request from my side. It is more personal view on how to debug.

Reply:
```std::endl``` was just a remnant of the past, it can be of course removed, @rdelliga will add a commit for the code cleaning you suggested